src/components/algolia-search/algolia-search.module.css

```diff
@@ -0,0 +1,181 @@
+.root {
+  position: relative;
+  margin-bottom: 34px;
+}
+
+.form {
+  border: none;
+  box-shadow: var(--token-surface-inset-box-shadow);
+  border-radius: 5px;
+  position: relative;
+  display: flex;
+  flex-direction: row;
+
+  &:focus-within {
```

Comment:
We want the focus outline on the full visual element, which is `.form` when there is focus on the input or buttons within.